drv/cosmo-hf/src/hf.rs

```diff
@@ -339,6 +354,20 @@ impl idl::InOrderHostFlashImpl for ServerImpl {
         Ok(())
     }
 
+    fn sector_erase_dev(
+        &mut self,
+        msg: &RecvMessage,
+        dev: HfDevSelect,
+        addr: u32,
+        protect: HfProtectMode,
+    ) -> Result<(), RequestError<HfError>> {
+        let prev = self.dev;
+        self.set_dev(msg, dev)?;
+        let r = self.sector_erase(msg, addr, protect);
+        self.set_dev(msg, prev).unwrap();
```

Comment:
Should this be `unwrap_lite()`. What is the difference these days?

Reply:
They do the same thing, but `unwrap_lite()` doesn't add as much code because it skips formatting the backtrace.  We use one or the other depending on vibes (e.g. if we're trying to keep a task under a particular size boundary).